decidim-proposals/app/controllers/concerns/decidim/proposals/orderable.rb

```diff
@@ -16,18 +16,51 @@ module Orderable
         # Available orders based on enabled settings
         def available_orders
           @available_orders ||= begin
-            available_orders = %w(random recent)
-            available_orders << "most_voted" if most_voted_order_available?
-            available_orders << "most_endorsed" if current_settings.endorsements_enabled?
-            available_orders << "most_commented" if component_settings.comments_enabled?
-            available_orders << "most_followed" << "with_more_authors"
+            available_orders = [default_order]
+            available_orders.concat(possible_orders.reject { |order| order == default_order })
```

Comment:
In Rails 6 we also now have `array.excluding(*elements)`:
https://api.rubyonrails.org/v6.1.4/classes/Array.html#method-i-excluding

More simply, this could be written as:
```ruby
[default_order] + possible_orders.excluding(default_order)
```